libs/sdk-backend-tiger/src/backend/workspace/catalog/rsqlFilter.ts

```diff
@@ -0,0 +1,50 @@
+// (C) 2021 GoodData Corporation
+import { isUriRef, ObjRef } from "@gooddata/sdk-model";
+import isEmpty from "lodash/isEmpty";
+import { MetadataGetEntitiesParams } from "@gooddata/api-client-tiger";
+import invariant from "ts-invariant";
+
+export function tagsToRsqlFilter({
+    includeTags,
+    excludeTags,
+}: {
+    includeTags: ObjRef[];
+    excludeTags: ObjRef[];
+}): string[] {
+    const rsqlFilterParts = [];
+
+    if (!isEmpty(includeTags)) {
+        const includeTagsIdentifiers = tagsToIdentifiers(includeTags);
+        rsqlFilterParts.push(`tags=in=(${includeTagsIdentifiers.join(",")})`);
+    }
+
+    if (!isEmpty(excludeTags)) {
```

Comment:
Can excludeTags work in combination with includeTags on tiger? (I think that on bear, it's not possible, so maybe we should mention it on SPI level - that possibility to combine both options depends on the backend implementation)?

Reply:
I have tried to use both in one request and it works. But I did not check it with BE developers